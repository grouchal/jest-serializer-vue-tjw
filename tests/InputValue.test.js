const helpers = require('./helpers.js');

import { shallowMount } from '@vue/test-utils';
import InputValue from './components/InputValue.vue';

describe('InputValue.vue', () => {
  test('Value shown in snapshot', () => {
    helpers.mockSettings({});

    const wrapper = shallowMount(InputValue);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
