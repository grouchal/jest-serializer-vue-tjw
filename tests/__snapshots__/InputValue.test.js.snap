// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputValue.vue Value shown in snapshot 1`] = `
<div>
  <input title="text" value="'There is a title attribute on this input'">
  <input value="'There are no other attrubutes on this input'">
  <input value="Native value with attribute" title="text">
  <input value="Native value without attribute">
  <input value="['moo']">
  <input type="checkbox">
  <input type="range" value="75">
  <input value="{a:{c:true},b:{d:'text'}}">
</div>
`;
